### ISSUE TITLE
Checkbox/RadioButton: No support for selecting option with keyboard

### ISSUE THREAD

**Issue by harrybradshaw** [primefaces/primereact#3221]
### Describe the bug

Whilst can tab through and focus on options, there is no way to simulate a click with the keyboard.

### Reproducer

https://codesandbox.io/s/suspicious-mountain-k10lhx

### PrimeReact version

8.4.0

### React version

18.x

### Language

ALL

### Build / Runtime

Create React App (CRA)

### Browser(s)

Chrome

### Steps to reproduce the behavior

1. Click on label to focus on button.
2. No key will cause checkbox to checked.

### Expected behavior

Enter key should be able to perform same behaviour as click.

**Comment by melloware** [comment]
This will be addressed when all the accessibility issued are going to be addressed at the same time.
